db/migrate/20150525081931_remove_duplicate_tokens.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveDuplicateTokens < ActiveRecord::Migration
+  def up
+    existing_tokens = Host::Managed.joins(:token).map(&:token).compact
+    if existing_tokens.empty?
+      Token.delete_all
+    else
+      Token.where('id not in (?)', existing_tokens).delete_all
+    end
+
+    remove_index :tokens, :host_id
```

Comment:
I think this is failing on MySQL if it's trying to remote an index that might not exist.